clang/include/clang/AST/ASTContext.h

```diff
@@ -1726,6 +1726,12 @@ class ASTContext : public RefCountedBase<ASTContext> {
 
   QualType getEnumType(const EnumDecl *Decl) const;
 
+  /// Compute BestType and BestPromotionType for an enum based on the highest
+  /// number of negative and positive bits of its elements.
+  bool computeBestEnumTypes(bool isPacked, unsigned NumNegativeBits,
```

Comment:
nit: maybe also mention that it returns `true` if there is no suitable integer type.